tests/unit/ConnectionTest.php

```diff
@@ -461,14 +461,14 @@ public function testDSNConfig()
         $this->assertEquals('mysql:host=127.0.0.1;dbname=yiitest', $connection->dsn);
 
         unset($dsn['driver']);
-        $this->expectException('yii\base\InvalidConfigException');
+        $this->expectException('yii\exceptions\InvalidConfigException');
         $connection = new Connection(['dsn' => $dsn]);
     }
 
     public function testServerStatusCacheWorks()
     {
-        $cache = new Cache(['handler' => new ArrayCache()]);
-        Yii::getApp()->set('cache', $cache);
+        $cache = new Cache(new ArrayCache());
+        $this->container->set('cache', $cache);
```

Comment:
@hiqsol is this correct ? Previously this was setting `cache` in the Application, but now it's doing it on `$this`
(yes, I'm still confused 😅 )